File: C/Condicional/Troco.c
#include <stdio.h>
int main()
{
int Qnt;
double P, Din, Troco;

printf("Preço unitario do produto:");
scanf("%lf", &P);
printf("Quantidade comprada:");
scanf("%d", &Qnt);
printf("Dinheiro Recebido: ");
scanf("%lf", &Din);

Troco = Din - (P * Qnt);
    if(Troco<0){
        Troco = Troco * -1;
    printf("Dinheiro Insuficiente. Faltam %.2lf Reais", Troco);
}else {
printf("Troco = R$ %.2lf", Troco);

}




}

File: Phyton/Sequencial/Troco.py
preco = float(input("Preço unitário do produto: "))
dinheiro = int(input("Quantidade comprada: "))
recebido = float(input("Dinheiro recebido: "))

troco = recebido - (preco*dinheiro)

print(f"TROCO = {troco:.2f}")